app/controllers/sections/importInformation/ImportCustomsOfficeCodeController.scala

```diff
@@ -0,0 +1,94 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package controllers.sections.importInformation
+
+import controllers.BaseNavigationController
+import controllers.actions._
+import forms.sections.importInformation.ImportCustomsOfficeCodeFormProvider
+import models.requests.DataRequest
+import models.response.InvalidUserTypeException
+import models.{GreatBritainRegisteredConsignor, Mode, NorthernIrelandRegisteredConsignor, UserType}
+import navigation.ImportInformationNavigator
+import pages.DispatchPlacePage
+import pages.sections.importInformation.ImportCustomsOfficeCodePage
+import play.api.i18n.MessagesApi
+import play.api.mvc.{Action, AnyContent, MessagesControllerComponents}
+import services.UserAnswersService
+import views.html.sections.importInformation.ImportCustomsOfficeCodeView
+
+import javax.inject.Inject
+import scala.concurrent.Future
+
+class ImportCustomsOfficeCodeController @Inject()(override val messagesApi: MessagesApi,
+                                                  override val auth: AuthAction,
+                                                  override val userAllowList: UserAllowListAction,
+                                                  override val getData: DataRetrievalAction,
+                                                  override val requireData: DataRequiredAction,
+                                                  override val navigator: ImportInformationNavigator,
+                                                  override val userAnswersService: UserAnswersService,
+                                                  formProvider: ImportCustomsOfficeCodeFormProvider,
+                                                  val controllerComponents: MessagesControllerComponents,
+                                                  view: ImportCustomsOfficeCodeView
+                                                 ) extends BaseNavigationController with AuthActionHelper {
+
+  def onPageLoad(ern: String, draftId: String, mode: Mode): Action[AnyContent] =
+    authorisedDataRequestAsync(ern, draftId) {
+      implicit request =>
+        withAnswer(DispatchPlacePage) {
```

Comment:
Neither XIRC nor GBRC get asked the Dispatch Place question. Can you double check with the UCD folk if the text in the ticket is actually correct? I don't see any mention of the Dispatch Place in the copydeck, it only says "if XIRC then text is ABC, if GBRC then text is DEF".